addons/bluguide/bluguide.lua

```diff
@@ -6,7 +6,7 @@ traitboxes = require("ui/traitboxes")
 spellboxes = require("ui/spellboxes")
 setspells = require("masterlist")
 
-_addon.version = '1.1'
+_addon.version = '1.2 Aug 28th, 2017'
```

Comment:
Would appreciate it if you could keep the versioning scheme of the original author.

Reply:
Ah ok I was trying to change it so it more noticeable. All these addons update and most people don't notice because its just a bump from 1.x to 1.y but will do